data/json/items/comestibles.json

```diff
@@ -2620,8 +2620,8 @@
         "price" : 500,
         "material" : "flesh",
         "volume" : 1,
-        "charges" : 1,
-        "stack_size" : 2,
+        "stack_size" : 4,
```

Comment:
To maintain its dimensions.

Before:
1 charge per unit, 2 units consisting of 2 charges per 1 volume
After:
2 charges per unit, 2 units consisting of 4 charges per 1 volume